README.MD

```diff
@@ -22,7 +22,7 @@ Presidio can be integrated into any data pipeline for intelligent PII scrubbing.
 
 ## Demo
 
-[Try Presidio with your own data](https://presidio-demo.westeurope.cloudapp.azure.com/)
+[Try Presidio with your own data](https://presidio-demo-dev.azurewebsites.net/)
```

Comment:
```suggestion
[Try Presidio with your own data](https://presidio-demo.azurewebsites.net/)
```

This will be the link in the next few daya

Reply:
It's temporary for now. We'll change it once prod is ready